localisation/000_wc_traits.csv

```diff
@@ -622,3 +622,9 @@ proud_desc;For some reason this character is proud or even selfish. Proud charac
 humble_desc;This character is humble and [From.GetHerHis] trait is expressed in moderation in [From.GetHerHis] demands, indifference to wealth and luxury, as well as [From.GetHerHis] determination to show [From.GetHerHis]self. Humble people tend to dislike proud ones.;;Dieser Charakter ist bescheiden und [From.GetHerHis] Charakterzug dr�ckt sich in [From.GetHerHis]en Forderungen aus, in Gleichg�ltigkeit gegen�ber Reichtum und Luxus sowie in [From.GetHerHis]er Entschlossenheit, sich selbst zu zeigen. Bescheidene Leute neigen dazu, Stolze nicht zu m�gen.;;;;;;;;;;;x
 pneumonic_desc;This character suffers from pneumonia � a deadly infection of the lungs.;;Dieser Charakter leidet an einer Lungenentz�ndung, einer oftmals t�dlichen Infektion.;;;;;;;;;;;x
 lefthanded_desc;This character is left-handed. People who are left-handed tend to have a considerable advantage in combat.;;Dieser Charakter ist Linksh�nder. Personen, die Linksh�nder sind, haben im Kampf oft einen erheblichen Vorteil.;;;;;;;;;;;x
+creature_wickerman;Wickerman;;Weidenmann;;;;;;;;;;;x
+creature_wickerman_desc;This character is a wickerman. Wickermen are created by Throsic covens from inanimate organic materials such as wood, flesh, fur, and bones and are animated by ancient Drust death magic. They are powered by the souls of mortal victims or by the spirits of ancient Drust and thus give the spirits physical form.;;Dieser Charakter ist ein Weidenmann. Weidenm�nner werden von throsischen Zirkeln aus unbelebten organischen Materialien wie Holz, Fleisch, Fell und Knochen erschaffen und durch die antike Todesmagie der Drust animiert. Sie werden von den Seelen der sterblichen Opfer oder von den Geistern der antiken Drust angetrieben und geben den Geistern somit eine physische Form.;;;;;;;;;;;x
```

Comment:
```suggestion
creature_wickerman_desc;This character is a wickerman. Wickermen are created by Throsic covens from inanimate organic materials such as wood, flesh, fur, and bones and are animated by ancient Drust death magic. They are powered by the souls of mortal victims or the spirits of ancient Drust and thus give the spirits physical form.;;Dieser Charakter ist ein Weidenmann. Weidenm�nner werden von throsischen Zirkeln aus unbelebten organischen Materialien wie Holz, Fleisch, Fell und Knochen erschaffen und durch die antike Todesmagie der Drust animiert. Sie werden von den Seelen der sterblichen Opfer oder von den Geistern der antiken Drust angetrieben und geben den Geistern somit eine physische Form.;;;;;;;;;;;x
```
Removed the 'by' in "by the spirits of ancient Drust and"
Should I make the change in the other wicker desc?
Suggestion for a modified sentence, it's shorter, but not ultimately not required:
They are powered by the souls of mortal victims or the spirits of ancient Drust, giving the spirits physical form.

Reply:
I think, the "by" should stay here. The constructs are animated/powered by the magic, as I understood it.

Reply:
'powered by' either the souls of mortal victims or the spirits
the 2nd by is redundant and repetitive, unless they're both done through separate methods, then it should be specified

Reply:
Again, I'm not sure it's wrong. When the wickerman is created, it gets animated/resurrected by drust MAGIC. Once it's walking around, it's powered by either souls of mortal victims or by spirits of ancient drusts... Like, the magic is what's starting it, but you need a soul/spirit to keep it running

Reply:
changed, need German corrections.